server/tests/UserControllerTest.js

```diff
@@ -0,0 +1,27 @@
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+import app from '../app';
+
+chai.use(chaiHttp);
+const should = chai.should();
+
+describe('testing signup feature', () => {
+  it('should successfully signup a user that enters all required fields correctly', (done) => {
+    const newUser = {
+      firstName: 'Augustine',
+      lastName: 'ezinwa',
+      email: 'jet55591a@gmail.com',
+      password: '5654545qa',
+      confirmPassword: '5654545qa',
+    };
+    chai.request(app).post('/api/v1/auth/signup')
```

Comment:
This test should assert that a new user was actually inserted into the database. This will ensure that the core functionality is actually tested.